src/uint256.h

```diff
@@ -91,6 +91,15 @@ class base_blob
                ((uint64_t)ptr[7]) << 56;
     }
 
+    uint32_t GetUint32LE(int pos) const
```

Comment:
You can use `ReadBE32` for this, it's more efficient.

If instead this would be BE (`ReadBE32` also exists), you can get rid of `Uint32ToUint8VectorLE` and instead print using `strprintf("%08x", fingerprint)`.